src/users-api/src/main/java/engineering/everest/lhotse/users/domain/commands/UpdateUserRolesCommand.java

```diff
@@ -0,0 +1,26 @@
+package engineering.everest.lhotse.users.domain.commands;
+
+import java.io.Serializable;
+import java.util.Set;
+import java.util.UUID;
+
+import javax.validation.constraints.NotNull;
+
+import org.axonframework.modelling.command.TargetAggregateIdentifier;
+
+import engineering.everest.lhotse.axon.common.domain.Role;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class UpdateUserRolesCommand implements Serializable {
+    @TargetAggregateIdentifier
+    private UUID userId;
+    private Set<Role> roles;
+
+    @NotNull
```

Comment:
I have checked it and the command validation is happening.

Requests are not reaching to a command handler when a command is invalid.

```
public class DeleteAndForgetUserCommand implements ValidatableCommand {
    @TargetAggregateIdentifier
    private UUID userId;
    @NotNull
    private UUID requestingUserId;
    @NotBlank
    private String requestReason;
}
```
For example in case of the above command, if the requestReason is blank then we are getting the response `"message": "requestReason: must not be blank"`.

Reply:
Nice, but a little dangerous as such checks should, conceptually, be implemented in the aggregate so the business rules are explicitly modeled. 

Thanks for verifying!